cooltools/api/dotfinder.py

```diff
@@ -1275,16 +1148,16 @@ def clustering_step(
 
 # consider switching step names - "extraction" to "FDR-thresholding"
 # and "thresholding" to "enrichment" - for final enrichment selection:
-def thresholding_step(centroids, obs_raw_name=observed_count_name):
-    # (2)
-    # filter by FDR, enrichment etc:
-    enrichment_factor_1 = 1.5
-    enrichment_factor_2 = 1.75
-    enrichment_factor_3 = 2.0
-    FDR_orphan_threshold = 0.02
-    ######################################################################
-    # # Temporarily remove orphans filtering, until q-vals are calculated:
-    ######################################################################
+def cluster_filtering_hiccups(
+    centroids,
+    obs_raw_name=observed_count_name,
+    enrichment_factor_1=1.5,
+    enrichment_factor_2=1.75,
+    enrichment_factor_3=2.0,
+    FDR_orphan_threshold=0.02,
+):
+    # ad hoc filtering by enrichment, FDR for singletons etc.
```

Comment:
ad hoc filtering of centroids by enrichment relative to recommended kernels, 

(just thinking to note somehow that this depends on the recommend_kernels)

Reply:
added doctstring to this and renamed vars